### DISCUSSION TITLE
Willing to contribute to the Apollo-Theme

### DISCUSSION THREAD

**Discussion by GregOnNet** [primefaces/community#1579] [PrimeNG Templates] [answered]
Hi,

my company uses the Apollo theme.
The code base is quite complex, and it seems there a some edge cases concerning routing and navigation.

We are wondering if there is a way we can contribute to Apollo:
- Refactoring the code base
- Adding Tests ensuring the template works as expected
- Using the latest features of Angular to simplify the code even more (Standalone APIs, Signals, Signal-Based in- and outputs)

**Comment by GregOnNet** [comment]
There is another discussion that contains several issues while adopting the theme: https://github.com/orgs/primefaces/discussions/1571

**Comment by GregOnNet** [comment]
cc @cagataycivici

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

We are grateful for your willingness to help us improve Apollo. However, since Apollo is not an open-source project and is a commercial product, we unfortunately cannot accept this. On the other hand, you can open a discussion about the improvements you would like to see and inform us. We review and consider issue reports, enhancement & new feature requests from our users while preparing updates for each major version.

**Reply by GregOnNet** [reply]
Hi, thanks for the response.
